### PR TITLE
add da.gd

### PR DESCRIPTION
**Pull request by relrod**
I couldn't get the tests to run, so hold of merging until Travis passes.

### CODE REVIEW

File: pyshorteners/shorteners/__init__.py [hunk 1]
```diff
@@ -72,7 +74,7 @@ def __init__(self, engine=Shorteners.SIMPLE, **kwargs):
             except AttributeError:
                 raise UnknownShortenerException(
                     'Please enter a valid shortener.'
-                    ' {} class does not exists'.format(self.engine)
+                    ' {} class does not exist'.format(self.engine)
```
Comment (ellisonleao) [inline pyshorteners/shorteners/__init__.py]: i believe the correct spelling is `exists` . Can you revert this change, please?

Comment (relrod) [inline pyshorteners/shorteners/__init__.py]: "class" is singular, so "exist" is grammatically correct. Do you still want me to revert it?

**Comment by relrod** [comment]
@ellisonleao I'm not sure why the tests are getting a connection refused to da.gd, when I test it manually it works fine. Am I missing something somewhere? :(

**Comment by relrod** [comment]
(Also, I'll squash these into one commit before you merge, once the tests are passing)

File: pyshorteners/shorteners/__init__.py (hunk 1 above)
Comment (ellisonleao) [inline pyshorteners/shorteners/__init__.py]: ok, let's leave it that way.

**Comment by ellisonleao** [comment]
Any updates on this one @relrod ?

**Comment by ellisonleao** [comment]
Hey @relrod i will try to fix the tests on my side, thanks for the PR anyway!
